cpp/include/Ice/SSL/ClientAuthenticationOptions.h

```diff
@@ -73,7 +73,7 @@ namespace Ice::SSL
         /// cipher suite.
         /// @param info The connection info object that provides additional connection-related data. The
         /// `ConnectionInfo` type is an alias for the platform-specific connection info class.
-        /// @return true if the certificate chain is valid and the connection should proceed; false if the certificate
+        /// @return `true` if the certificate chain is valid and the connection should proceed; false if the certificate
```

Comment:
Again, there are probably even more in this file that are being missed.
This regex should find them: ```//.*(?<!`)(?:true|false)(?!`)```

```suggestion
        /// @return `true` if the certificate chain is valid and the connection should proceed; `false` if the certificate
```